docs/Introduction.rst

```diff
@@ -3,31 +3,11 @@ Introduction
 ************
 
 The Unified Post Processor (UPP) software package is a software package designed to generate useful
```

Comment:
Recommend removing the first instance of "software package" and instead just "The Unified Post Processor (UPP) is a software package designed...."

Reply:
haha - fixed